src/common/Utilities/SFMTRand.cpp

```diff
@@ -20,12 +20,39 @@
 #include <array>
 #include <functional>
 #include <random>
+#include <ctime>
+
 #if defined(__aarch64__)
-#include <mm_malloc.h>
+    #if defined(__clang__)
+        #include <mm_malloc.h>
+    #elif defined(__GNUC__)
+        static __inline__ void *__attribute__((__always_inline__, __nodebug__, __malloc__))
+        _mm_malloc(size_t __size, size_t __align) {
```

Comment:
TC C++ codestyle: put opening curly brackets ` { ` on a new line (see lower part of the file for examples)